ext/telemetry/lib.rs

```diff
@@ -1,12 +1,15 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
 
+#![allow(clippy::too_many_arguments)]
```

Comment:
Do we have to do it for the whole module? :(

Reply:
It's due to a bug in core - Divy already knows about it

Reply:
https://github.com/denoland/deno_core/pull/1028